deploy/deploy-systemd.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+set -e
+
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+BRANCH=$1
+name=
+
+if [ "x$BRANCH" = "x" ]; then
+  echo "Please pass branch to deploy as first argument"
+  exit 1
+fi
+
+if [ "x$BRANCH" = "xproduction" ]; then
+  name=foodsaving-world
+else
+  name=foodsaving-world-dev
+fi
+
+function restart-workers() {
+  sudo systemctl restart "${name}-worker.target"
+}
+
+backend_dir="/var/www/${name}/www"
```

Comment:
Yes, I did intend it. To make it consistent with all the other apps on the server. Good point for the cronjobs. Although perhaps we could go all-out on the systemd, using [timers](https://wiki.archlinux.org/index.php/Systemd/Timers#As_a_cron_replacement).